src/assets/components/shared/SearchBar/styles.ts

```diff
@@ -0,0 +1,85 @@
+import { borderRadius, colors } from "../../../../colors";
+
+const textFieldBaseStyles = {
+  baseStyle: {
+    width: "400px",
+
+    "& .MuiInputBase-root.MuiInput-root": {
+      borderColor: "none",
+    },
+    "& .MuiInputBase-root": {
+      borderRadius: borderRadius._1,
+    },
+  },
+};
```

Comment:
Remove the baseStyles key as its redundant... 

```suggestion
const textFieldBaseStyles = {
    width: "400px",

    "& .MuiInputBase-root.MuiInput-root": {
      borderColor: "none",
    },
    "& .MuiInputBase-root": {
      borderRadius: borderRadius._1,
    },
  },
```